language-support/ts/daml-ledger/index.test.ts

```diff
@@ -153,7 +153,7 @@ describe("streamSubmit", () => {
     stream.on("change", mockChange);
     const restoreConsole = mockConsole();
     mockInstance.serverSend({ warnings: ["oh oh"] });
-    expect(console.warn).toHaveBeenCalledWith("Ledger.streamQueries warnings", {"warnings": ["oh oh"]});
+    expect(console.warn).toHaveBeenCalledWith("QueryStreamsManager warnings", {"warnings": ["oh oh"]});
```

Comment:
The point here was to contain the function name the user actually called, so I'd have a slight preference for not changing the log messages if possible. QueryStreamsManager is an implementation detail the user should not care about, presumably.

Reply:
Initially I was aiming to do this but decided to keep it simple as I didn't think it was too much of an issue, but this makes sense. I'll do it, thanks for the feedback.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/10221/commits/0811eff4dc794a2f0ce12bd06ca85f40aef5cbbe

This also restores what would have been the previous behavior of having one log line for every outstanding query. Since (so far) I tried to make sure that the behavior of virtual streams is identical to the existing one backed by an actual web socket, this is a welcome improvement. 🙇🏻